hologram_dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+## Preparation
+    1.  Clone the repository 
+        git clone https://github.com/googleinterns/sarahchen-intern-2020.git
+    2.  Navigate into the "hologram_dashboard" folder
+        cd hologram_dashboard
+
+## Server 
+Follow the following steps to run the server 
+
+    1.  Install bazel by following the steps of 
+        https://docs.bazel.build/versions/3.2.0/install.html
+        for your specific system eg. ubuntu
+    2.  Build the server from the hologram_dashboard folder
+        bazel build //server
```

Comment:
I think it should be 

bazel build //server:server

in order to build the target binary.